Dockerfile

```diff
@@ -1,21 +1,28 @@
-FROM debian:jessie
+FROM debian:stretch
 
 ENV MLAT yes
 
-RUN apt-get -y update && apt-get -y upgrade && apt-get -y dist-upgrade
+RUN apt update && apt install -y \
+  autoconf \
+  build-essential \
+  debhelper \
+  dh-systemd \
+  git \
+  libz-dev \
+  python3-dev \
+  python3-venv \
+  socat \
+  tcl8.6-dev \
+  wget
 
-RUN apt-get -y install git
 RUN git clone https://github.com/flightaware/piaware_builder.git
-
-RUN apt-get -y install wget build-essential debhelper tcl8.6-dev autoconf python3-dev python3-venv libz-dev dh-systemd
-RUN piaware_builder/sensible-build.sh jessie
-WORKDIR /piaware_builder/package-jessie
+RUN piaware_builder/sensible-build.sh stretch
+WORKDIR /piaware_builder/package-stretch
 RUN dpkg-buildpackage -b
-RUN apt-get -y install net-tools tcllib tcl-tls itcl3 tclx8.4 init-system-helpers
```

Comment:
"apt install" handles installing the package's dependencies for us. I have tested it and it does work.

This functionality has existed since [1.1~exp1](http://metadata.ftp-master.debian.org/changelogs/main/a/apt/stable_changelog), look for "apt-get install foo_1.0_all.deb". I believe it's from [this commit](https://salsa.debian.org/apt-team/apt/commit/00026265672f2663e0aa0ab1a3fa7333f718a643#01ed7b9897d479cd199b08abb60078324e03bfb2).

Looks like this feature isn't correctly documented. From https://manpages.debian.org/stretch/apt/apt-get.8.en.html you would think this wouldn't work.
> install pkg(s)
> This option is followed by one or more packages desired for installation. Each package is a package name, **not a fully qualified filename** (for instance, in a Fedora Core system, glibc would be the argument provided, not glibc-2.4.8.i686.rpm).

Reply:
This [bug update](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=825467#10) over in the APT issue tracker sums this new behavior well.

I've [filed a new bug](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=911728) with the APT developers to correct the wording on the man page.